test/runtests.jl

```diff
@@ -341,7 +341,7 @@ end
 let weakdict = pyimport("weakref")["WeakValueDictionary"]
     # (use weakdict for the value, since Python supports
     #  weak references to type objects)
-    @test weakdict(Dict(3=>weakdict)) == Dict(3=>weakdict)
+    @test weakdict(Dict(3=>weakdict))[:__getitem__](3) == weakdict
```

Comment:
Probably:
```jl
@test Dict(weakdict(Dict(3=>weakdict))) == Dict(3=>weakdict)
@test weakdict(Dict(3=>weakdict))[3] == weakdict
```
The first test would have caught the PyPlot problem, and for the second test I think you can just do `[3]` rather than calling `__getitem__`?